ui/modules/piggyback/piggyback.html

```diff
@@ -0,0 +1,103 @@
+                                        Insert Template
+                                    </button>
+                                </div>
+                            </div>
+                            <div class="col-md-6 resizable_flex_column">
+                                <div class="col-md-auto">
+                                    <button class="btn btn-outline-blue btn-sm button_round_left" style="float: right;"
+                                            id="buttonSubmit">
+                                        Submit
+                                    </button>
+                                    <button hidden class="btn btn-outline-secondary btn-sm button_round_left"
+                                            style="float: right;" id="buttonCancel">
+                                        Cancel
+                                    </button>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="input-group input-group-sm mb-1 has-validation">
+                            <label class="input-group-text">Service</label>
+                            <select class="form-select-sm" id="serviceSelector"
+                                    style="margin-top: auto; margin-bottom: auto;"></select>
+                            <button class="btn btn-outline-primary btn-sm button_round_left"
+                                    id="buttonLoadServiceInstances">
+                                <i class="bi bi-arrow-clockwise"></i>
+                            </button>
+                            <div class="invalid-feedback"></div>
+                        </div>
+
+                        <div class="input-group input-group-sm mb-1" hidden id="instanceContainer">
+                            <a class="nav-link active">Instance:</a>
+                            <select class="form-select-sm" id="instanceSelector"
+                                    style="margin-top: auto; margin-bottom: auto;"></select>
+                        </div>
+
+                        <div class="input-group input-group-sm mb-1 has-validation" style="width: 20%;">
+                            <label class="input-group-text">Timeout in sec</label>
+                            <input type="text" class="form-control form-control-sm" id="timeout"></input>
+                            <div class="invalid-feedback"></div>
+                        </div>
+                        <div class="input-group input-group-sm mb-1 has-validation">
+                            <label class="input-group-text">Actor path</label>
+                            <input type="text" class="form-control form-control-sm" id="targetActorSelection"></input>
+                            <div class="invalid-feedback"></div>
+                        </div>
+                        <div class="has-validation">
+                            <h6 id="headerValidationElement">Header</h6>
```

Comment:
Just to be consistent to other places with validation above tables, we can use a hidden input field as a validation element instead of using the headline.